schemas/com.snowplowanalytics.snowplow/weather_enrichment_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,52 @@
+{
+	"$schema": "http://json-schema.org/schema#",
+	"description": "Schema for weather enrichment",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "weather_enrichment_config",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+	"properties": {
+		"vendor": {
+			"type": "string"
+		},
+		"name": {
+			"type": "string"
+		},
+		"enabled": {
+			"type": "boolean"
+		},
+		"parameters": {
+			"type": "object",
+			"properties": {
+				"apiKey": {
+                    "type": "string"
+				},
+                "cacheSize": {
+                    "type": "integer",
+                    "minimum": 1,
+                    "maximum": 1000000
+                },
+                "rounder": {
```

Comment:
It rounds geo coordinates to part of 1.
For example, rounder 2 will round 11.6 to 11.5, 11.1 to 11.0 (so round it to 1/2); rounder 5 will round 3.312 to 3.2 (so round it to 1/5).
This is how our cache works for geo coordinates. Greater "rounder" - more accurate cache.